node/sc/kas/anchor.go

```diff
@@ -126,16 +126,16 @@ func (anchor *Anchor) AnchorBlock(block *types.Block) error {
 	payload := dataToPayload(anchorData)
 
 	res, err := anchor.sendRequest(payload)
-	if err != nil {
+	if err != nil || res.Code != codeOK {
+		if res != nil {
```

Comment:
I think if `res` is nil, then it will panic before 129 line.

Reply:
If err is not nil, `res.Code != codeOK` will not be checked, so There is no case to access `nil` `res`'s `Code`.

Reply:
Oh, I got it. Thanks.